src/hotspot/share/gc/shared/memAllocator.hpp

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2018, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2018, 2019 Oracle and/or its affiliates. All rights reserved.
```

Comment:
Do not change copyright declaration of Oracle

Reply:
it used to be a convention to update field 'year' to current, not a problem